apps/showcase/project.json

```diff
@@ -51,7 +51,7 @@
       "executor": "@angular-devkit/build-angular:application",
       "inputs": [
         "source",
-        "^source-and-structure",
+        "^structure",
```

Comment:
I've changed that to match the recent refactor of Nx inputs (`structure-only` is now `dist/structure` and `structure` is `structure-only`+`source`)
check `nx.json` and https://github.com/AmadeusITGroup/otter/pull/2338